nomos-services/data-availability/indexer/src/indexer/mod.rs

```diff
@@ -0,0 +1,23 @@
+use nomos_core::da::certificate::{metadata::Metadata, vid::VID};
+
+use crate::storage::DaStorageAdapter;
+
+pub struct Range<Index> {
+    pub from: Index,
+    pub to: Index,
+}
```

Comment:
We can use the rust std range: https://doc.rust-lang.org/std/ops/struct.Range.html (?)